templates/web/base/reports/index.html

```diff
@@ -139,4 +139,58 @@ <h2 class="dashboard-subheading">[% loc('Top 5 most used categories') %]</h2>
     </div>
 </div>
 
+[% IF display_contacts %]
+<div class="dashboard-row">
+    <div class="dashboard-item dashboard-item--12">
+        <h2 class="dashboard-subheading">[% tprintf( loc('Where we send %s reports'), body.name ) %]</h2>
+        [% IF body.send_method == 'Refused' %]
+          <p>
+            [% tprintf( loc('%s currently does not accept reports from FixMyStreet.'), body.name) %]
+          </p>
+
+          <p>
+            [% loc('If you&rsquo;d like to discuss this then <a href="/contact">get in touch</a>.') %]
+          </p>
+        [% ELSIF body.send_method == 'Noop' %]
+          <p>
+            [% tprintf( loc('Reports are currently not being sent to %s.'), body.name ) %]
+          </p>
+        [% ELSIF body.send_method != 'Email' AND body.send_method != ''  %]
+          <p>
+              [% tprintf( loc('Reports to %s are currently sent directly into backend services.'), body.name) %]
+          </p>
+        [% ELSE %]
+        <p>
+          [% loc('We currently send all reports to the email addresses below.') %]
+        </p>
+
+        <p>
+          [% loc('Did you know that if you used the approved open standard Open311 you could send reports directly into your own backend services &ndash; and get much more control over what additional information you request?') %]
+        </p>
+
+        <p>
+          [% loc('If that&rsquo;s new to you, <a href="https://www.mysociety.org/2013/01/10/open311-introduced/">take a look at our simple Open311 primer</a> to see what you need to do to get up and running in a few days.') %]
+        </p>
+
+        <p>
+          [% loc('If you would like to change either the categories or the contact emails below then <a href="/contact">get in touch</a>.') %]
+        <p>
+        <table class="dashboard-ranking-table">
+          <tr>
+            <th>[% loc('Category') %]</th>
+            <th>[% loc('Contact') %]</th>
+          </tr>
+          [% WHILE ( cat = contacts.next ) %]
+            <tr [% IF cat.state == 'deleted' %]class="is-deleted"[% END %]>
```

Comment:
This doesn't do anything here. Probably shouldn't show deleted contacts, loop though `live_contacts` rather than `contacts`?